src/main/webapp/view/transcript/transcript-seeker.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Seeks to a line in the transcript. */
+export default class TranscriptSeeker {
+  static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
+
+  #currentTranscriptLine;
+  #eventController;
+
+  /**
+   * Creates an instance of `TranscriptSeeker` for loading
+   * transcript lines onto the DOM.
```

Comment:
Why did you write: for loading transcript lines onto the DOM?

Is this a copy over from TranscriptArea?

Reply:
Sorry about that and thank you for catching it! I must have missed it. Updated.